src/components/comments/CommentList.jsx

```diff
@@ -17,17 +26,68 @@ export default function CommentList({ articleId }) {
     }
   }, [articleId]);
 
+  function handleChange(e) {
+    setCommentBody(e.target.value);
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    if (!user) {
+      console.log("User not logged in");
```

Comment:
lets remove the console.log as users will not be looking at the console. How about we throw an error here and display it for the user 🙂